sphinx/cmd/quickstart.py

```diff
@@ -347,6 +335,13 @@ def ask_user(d):
         d['master'] = do_prompt(__('Please enter a new file name, or rename the '
                                    'existing file and press Enter'), d['master'])
 
+    while not check_valid_dir(d):
```

Comment:
I understand all of `sep`, `dot`, `master` and `suffix` are missing just after asking root path. But this place is too far from question.
I think the goal of this PR is not to override `Makefile` and `make.bat` by default. To do that, how about changing `overwrite` option of `generate()` function to False?

Reply:
> But this place is too far from question.

What do you mean?

> I think the goal of this PR is not to override Makefile and make.bat by default. To do that, how about changing overwrite option of generate() function to False?

I don't know - but please feel free to do it like that.  While not perfect it would still be a good enough fix then I think.

You can push updates to my branch/this PR, or just create a new one and close this.

Reply:
>What do you mean?

What I meant is that the check for root path is going after 6 steps after from its question. I worried users might get confused.

>I don't know - but please feel free to do it like that. While not perfect it would still be a good enough fix then I think.

Okay, I'll reconsider about this later.